buildSrc/src/main/kotlin/Config.kt

```diff
@@ -0,0 +1,38 @@
+import org.gradle.api.JavaVersion
+
+object Config {
+
+    @JvmStatic
```

Comment:
i suppose this is necessary when used in groovy gradle files, so maybe its time to move to kotlin gradle?